packages/v/vid.stab/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : vid.stab
-version    : 1.1
-release    : 1
+version    : 1.1.1
+release    : 2
 source     :
-    - git|git://github.com/georgmartius/vid.stab.git : 2d82492533bf5004bc2c4f8213a1b622c45f6a44
+    - https://github.com/georgmartius/vid.stab/archive/refs/tags/v1.1.1.tar.gz : 9001b6df73933555e56deac19a0f225aae152abbc0e97dc70034814a1943f3d4
+homepage   : http://public.hronopik.de/vid.stab/
 license    : GPL-2.0
 component  : multimedia.library
 summary    : Video stabilization library
```

Comment:
The `%cmake` macros have generally been superseded by newer macros. Replace `%cmake .` with `%cmake_ninja`, `%make` with `%ninja_build`, and `%make_install` with `%ninja_install` (note that the make macros are only being changed because you're changing the `%cmake` macro.